tests/test_config_parsing/test_parser_error_collection.py

```diff
@@ -121,9 +121,17 @@ def assert_that_config_leads_to_error(
 
     with pytest.raises(ConfigValidationError) as caught_error:
         ErtConfig.from_file(config_filename)
+        # If the ert config did not raise any errors
+        # we manually raise an "empty" error to make
+        # this raise an assertion error that can be
+        # acted upon from assert_that_config_does_not_lead_to_error
+        raise ConfigValidationError(errors=[])
```

Comment:
this seems hacky / funky upon first read :grimacing: in a function called `asser_that_config_leads_to_error`, you would expect that the config file contents you pass in always lead to an error, otherwise you would use a different function. so why the need to artificially raise an error?

seems like it might cause false positives when using that function with a config file that one thinks should lead to an error, but it don't :thinking:

EDIT: i now see the catch in line 132. i guess it protects exactly against what i'm describing here :sweat_smile: